media/css/m24/components/navigation-refresh.scss

```diff
@@ -247,6 +262,20 @@
 
             .c-menu-panel {
                 transform: translateX(0);
+                height: 100vh;
+                overflow: visible auto;
+            }
+
+            #c-menu-panel-firefox {
+                max-height: 293px;
+            }
+
+            #c-menu-panel-about {
+                max-height: 338px;
+            }
+
+            #c-menu-panel-products {
+                max-height: 442px;
             }
```

Comment:
> I don't want them to take 100vh when it is desktop view.

Would setting them to 100vh only on mobile work? My worry is that keeping these values hard-coded as-is might risk leading to bugs in the future.

Reply:
I don't think so. The desktop height doesn't have to be `100h`, but it needs a value for the `overflow-y` auto scroll to work, but because the height of the panel is varied based on the content, we can potentially set a min-height to exceeds the tallest panel. But this will cause another issue that any panel shorter than the tallest one will have extra vertical space

Reply:
I think perhaps a `min-height` might be worth exploring. In the future, items in the nav could easily change, based on locale or things in the CMs even. I think we need to play for flexibility and try to avoid specific `max-height` values if at all possible.

Reply:
It occurred to me here that another option might be to instead show the mobile nav UX (with menu open / close button) at this breakpoint, instead of showing the desktop nav UI?

Reply:
It turns out that it is not super simple to change the breakpoint, since we have to override the existing nav styles. I am going to open a new PR to tackle the nav dropdown height issue

Reply:
Never mind. I figured out a one-line change to fix this :D